static/js/Misc.jsx

```diff
@@ -136,14 +179,21 @@ const ContentText = ({text, html, overrideLanguage, defaultToInterfaceOnBilingua
       return lang === langShort;
     });
   }
-  return renderedItems.map( x =>
-      isDangerouslySetInnerHTML ?
-          <span className={`contentSpan ${x[0]}`} lang={x[0]} key={x[0]} dangerouslySetInnerHTML={{__html: x[1]}}/>
-          :
-          <span className={`contentSpan ${x[0]}`} lang={x[0]} key={x[0]}>{x[1]}</span>
-  );
+  return renderedItems;
+}
+
+const ContentText = (props) => {
```

Comment:
You should direct people to read the docs of `ContentTextFilterByLang` to understand what the props are